tests/backends/data/test_fs.py

```diff
@@ -64,7 +64,7 @@ def test_backends_data_fs_data_backend_instantiation_with_settings(fs):
 
     try:
         FSDataBackend(settings)
-    except Exception as err:  # pylint:disable=broad-except
```

Comment:
don't we want this checked ?

Reply:
Unfortunately, it seems there is no equivalent rule for raising a `too broad except` in Ruff.
There is the `TRY` set of rules for `tryceratops` but it doesn't raise a warning for this, and we're far from following the good exception handling patterns (lots of warning raised) :grin:

Reply:
Adding `BLE` set of rules